normalize_repos/normalize_repos.py

```diff
@@ -26,10 +29,16 @@ def get_cc_repos(github):
     cc = get_cc_organization(github)
     return cc.get_repos()
 
```

Comment:
Use [Black](https://github.com/psf/black) to reformat the file. There should have been two blank lines before and after the function.

Reply:
Ok, got it.

Reply:
Format done.